build/__javaVersions.sh

```diff
@@ -1,4 +1,4 @@
 # This file was auto-generated from 'constants.yaml'. Changes may be overridden.
 
-JAVA_VERSION='17.0.2'
+JAVA_VERSION='11.0.14'
```

Comment:
Consider having variable names JAVA17_VERSION and JAVA11_VERSION, which is the latest minor/patch version for that major version.

Reply:
I've added JAVA11_VERSION but will keep JAVA_VERSION for now, we can use that as our LTS version